docusaurus/docs/learn/core-concepts/pki.md

```diff
@@ -1,89 +1,56 @@
 ---
 id: pki
-title: Public Key Infrastructure (PKI)
+title: Public Key Infrastructure
 sidebar_label: PKI
 ---
 
 import PkiTroubleshootingMd from '../../guides/05-troubleshooting/pki-troubleshooting.md'
 
-All Ziti Networks leverage [Public Key Infrastructure (PKI)](https://en.wikipedia.org/wiki/Public_key_infrastructure) to
-provide secure network connections.  This page is not intended to be a comprehensive guide. What it is, is a set of
-rules that must be followed to properly configure a Ziti Network. If there are issues when connecting any portion
-of a Ziti Network to another - this page should serve as a starting point of understanding.
+[Public Key Infrastructure](https://en.wikipedia.org/wiki/Public_key_infrastructure) (PKI) is used to establish trust between all Ziti components. The Ziti controller enrolls identities with a PKI that it manages, and externally-managed PKIs may be verified with the Ziti controller and used for enrollment. Enrollment is best suited for long-lived client certificate fingerprint authentication.
 
-:::note
-This article is about managing your own PKI. There's a guide for using public CA certificates.
-:::
+Alternatively to enrollment, external PKIs may be trusted to sign authentication instruments that are used to [claim an existing Ziti identity](./security/authentication/50-external-id-claims.md).
 
-The Ziti Network allows the operator to declare any trust anchors as valid. This means Ziti does not need to be
-configured with a full chain of certificates which link fully back to a root CA. A configuration using a full chain back
-to a root CA is of course supported but it is not explicitly required.  This allows the operator to configure a Ziti
-Network using one or more chains of trust back to the provided trust anchors.  The sections below will describe where
-these trust anchors can be configured.
+### Ziti Controller
 
-Ziti Network components are required to present a certificate to other Ziti Network components during the connection
-establishment. This certificate will need to be valid per the configured trust anchor store being connected to.
+The certificate authority (CA) in a Ziti network's PKI is the controller's edge enrollment CA. The edge enrollment CA issues client authentication certificates to identities during identity enrollment. The edge enrollment CA issues client and server authentication certificates to routers during router enrollment. 

```

Comment:
The first sentence, "The CA in a Ziti's network's PKI is the controller's edge enrollment CA" is superfluous and also technically incorrect. In any PKI there can be more than 1 CA.

I suggest rewording this to state that the Ziti Controller is configured with a CA that is used as the enrollment CA.

Reply:
https://github.com/openziti/ziti-doc/pull/592/commits/9fb0ef9913d8b2fd566a287e1f279c734297269c